### ISSUE TITLE
Has OSX Support gone now ?

### ISSUE THREAD

**Issue by n0x00** [SpiderLabs/Responder#50]
Hey guys, I just pulled down a new version of Responder I see it no longer works on Mac (Yosemite) 

It's failing to find interfaces

( en0 does exist)
``
[!] Error: en0: Interface not found
nulls-MacBook-Air:Responder null$ ``

`` en0: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	ether 28:cf:e9:4d:90:11 
	inet 192.168.1.107 netmask 0xffffff00 broadcast 192.168.1.255
	inet6 fe80::2acf:e9ff:fe4d:9011%en0 prefixlen 64 scopeid 0x4 
	nd6 options=1<PERFORMNUD>
	media: autoselect
	status: active
``

I'll have to go back to the metasploit modules huh ?

**Comment by ogoid** [comment]
It seems the current method of finding the interface ip address doesn't work on Macs. But `netifaces` Python package does provide a cross platform method. Just edit `utils.py`:

```Python
import netifaces
…
def FindLocalIP(Iface):
	if Iface == 'ALL':
		return '0.0.0.0'
	return netifaces.ifaddresses(Iface)[netifaces.AF_INET][0]['addr']

**Comment by lgandx** [comment]
Thanks for your report!
I'll implement the fix on the code soonish.
Cheers

**Comment by lgandx** [comment]
I'm not exactly sure when i'll have the time to fix this, so please use V2.1.4 (which supports osx) in the meantime: https://github.com/SpiderLabs/Responder/releases

**Comment by lgandx** [comment]
Problem solved, please use the latest release.
